src/Components/Homepage/Homepage.module.css

```diff
@@ -0,0 +1,76 @@
+.CanvaHomePage {
+  background-image: url("../../Media/canvaHomePage.png");
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  background-position: top right;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.createDesignBtn {
+  position: absolute;
+  right: 77px;
+  top: 13px;
+  height: 46px;
+  width: 158px;
+}
+
+@media (max-width: 1024px) {
+  .CanvaHomePage {
+    background-image: url("../../Media/laptopHomePage.png");
+    height: 100%;
+    width: 100%;
+    position: absolute;
+    background-position: top right;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+  .createDesignBtn {
+    position: absolute;
+    right: 87px;
+    top: 17px;
+    width: 150px;
+    height: 45px;
+  }
+}
+
+/* tablet screen size */
```

Comment:
out of interest: where did you find these screen sizes? might be useful to mention in the comment.

Reply:
The width breakpoints are from eyeing when the canva.com changes :D - put in the comment too